src/pages/new/docs/components/services/directoryScanner/index.vue

```diff
@@ -2,15 +2,17 @@
   <v-row>
     <v-col cols="12">
       <div v-html="markdown1" />
-      <v-img
-        src="@/assets/directory_scanner1.png"
-      />
+      <ng-img src="@/assets/new_service2.png" />
+
       <div v-html="markdown2" />
     </v-col>
   </v-row>
 </template>
 
 <script setup lang="ts">
+
+import ngImg from '@/components/ngImg/NgImg.vue'
```

Comment:
Ist nicht falsch, aber ich dränge darauf, dass wir in allen Frontends einheitlich sind. Deshalb werden selbstgeschriebene Vue Components immer mit großem Anfangsbuchstaben importiert und auch im template geschrieben, um sie einfacher von 3rd Party Components unterscheiden zu können

<NgImg src="" />

import NgImg from '@/...'